### DISCUSSION TITLE
Thankyou for banning me from your Tg TWICE!!

### DISCUSSION THREAD

**Discussion by BlueNecko** [ChefKissInc/NootedRed#276] [General]
@VisualEhrmanntraut both of my accounts were banned soon after joining and now I have no access to the group. This is totally not acceptable if this was an intended behaviour. acc: @Sameer_sahu98
![Screenshot 2024-06-15 at 1 26 28 AM](https://github.com/ChefKissInc/NootedRed/assets/66507691/5928bdb0-3846-4a7e-974d-ed8dbb00722d)

**Comment by VisualEhrmanntraut** [comment]
![809w1f](https://github.com/ChefKissInc/NootedRed/assets/30368284/7e0a98a4-9126-4fd9-a96b-6db448dbce31)

**Comment by BlueNecko** [comment]
By the way does forgedinvarient kext increases performance on Sonoma too ?? if so then cool I am going to add it. Nice work adding support for sequoia so fast

**Reply by VisualEhrmanntraut** [reply]
ForgedInvariant increases performance across the board when you have an AMD APU. It's a necessity, and it works better than the other TSC sync kexts.

**Comment by oceanondawave** [comment]
> @VisualEhrmanntraut both of my accounts were banned soon after joining and now I have no access to the group. This is totally not acceptable if this was an intended behaviour. acc: @Sameer_sahu98 ![Screenshot 2024-06-15 at 1 26 28 AM](https://private-user-images.githubusercontent.com/66507691/339967579-5928bdb0-3846-4a7e-974d-ed8dbb00722d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2Mjc4NTcsIm5iZiI6MTcxODYyNzU1NywicGF0aCI6Ii82NjUwNzY5MS8zMzk5Njc1NzktNTkyOGJkYjAtMzg0Ni00YTdlLTk3NGQtZWQ4ZGJiMDA3MjJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDEyMzIzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1YzA2ZjZmNTMxYjBjOWZmZThjOTg1ZDRjNTY0YmY5Yjg0ZDk0ZjE5ZWY3MGM5OGNkYjI2MmEzOWViZmEwYTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y3uGBzRKn2aJx-KIp7LmKwcqcm24o0vjLuZIWInAqhs)

Check this: https://github.com/ChefKissInc/NootedRed/discussions/223#discussioncomment-8900564.
